client/app/src/main/java/com/example/mrides/Notification/InboxAdapter.java

```diff
@@ -14,9 +15,11 @@
 public class InboxAdapter extends RecyclerView.Adapter<InboxAdapter.ViewHolder>{
 
     private Map<String,String> responseBody;
+    private static String inviteMessage;
 
-    public InboxActivity(Map<String,String> responseBody){
+    public InboxAdapter(Map<String,String> responseBody, String inviteMessage){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.